src/app/RuleManager.cpp

```diff
@@ -1,50 +1,30 @@
-#include <iostream>
-
 #include "RuleManager.h"
+#include "BasicRule.h"
+#include "FileWhitelist.h"
+#include "NetworkMonitor.h"
 
 namespace SAIL { namespace rule {
 
-RuleManager::RuleManager(const std::string &configPath) {
-    this->ctx = seccomp_init(SCMP_ACT_ALLOW);
-
+RuleManager::RuleManager(const std::string &configPath, const std::shared_ptr<util::Utils> &up) : up(up) {
+    this->ctxp.reset(new scmp_filter_ctx(seccomp_init(SCMP_ACT_ALLOW)));
     const YAML::Node config = YAML::LoadFile(configPath);
```

Comment:
configPath smells bad.
this constructor can't be mocked.
For example, in unit test, I wanna to build this object without a real file.

Reply:
fixed